assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -85,17 +85,28 @@ const CardTitle = ({
     if (!displayDate) {
       return null;
     }
-    return status === STORY_STATUS.PUBLISHED
-      ? sprintf(
-          /* translators: %s: last modified date */
+
+    switch (status) {
+      case STORY_STATUS.PUBLISH:
+        return sprintf(
+          /* translators: %s: published date */
           __('Published %s', 'web-stories'),
           displayDate
-        )
-      : sprintf(
+        );
+      case STORY_STATUS.FUTURE:
+        return sprintf(
+          /* translators: %s: future publish date */
```

Comment:
That is not accurate I think?

`displayDate` is still the last modified date as far as I can tell:

It says: "Scheduled 2 minutes ago"

<img width="480" alt="Screenshot 2020-07-09 at 10 12 30" src="https://user-images.githubusercontent.com/841956/87014673-bb6a0c80-c1cc-11ea-973a-da806071c57a.png">

I think it makes more sense to say "Scheduled for July 7, 2021"

Reply:
Good call out to fix this date here. This is in line with what the list view shows now. 


<img width="408" alt="Screen Shot 2020-07-09 at 9 52 59 AM" src="https://user-images.githubusercontent.com/10720454/87068143-0171a100-c1ca-11ea-88ff-a80ffe8acbc1.png">